main.js

```diff
@@ -0,0 +1,98 @@
+const { app, BrowserWindow, dialog, ipcMain } = require('electron');
+const path = require('path');
+const url = require('url');
+
+let mainWindow;
+
+app.commandLine.appendSwitch('autoplay-policy', 'no-user-gesture-required');
+
+function createWindow() {
+  // Create the browser window.
+  mainWindow = new BrowserWindow({
+    width: 1000,
+    height: 600,
+    icon: 'build/icon.png',
+    title: '** GAME TITLE GOES HERE **',
+    kiosk: false, // true for full screen, false for windowed
+    webPreferences: {
+      nodeIntegration: true,
+      contextIsolation: false,
+    },
+  });
+
+  // and load the index.html of the app.
+  mainWindow.loadURL(url.format({
+    pathname: path.join(__dirname, 'index.html'),
+    protocol: 'file:',
+    slashes: true,
+  }));
+
+  mainWindow.setMenu(null);
+
+  // -------------------------------------------------------
+  // uncomment this line if you need to debug startup issues
+  // -------------------------------------------------------
+  // mainWindow.webContents.openDevTools();
+  // -------------------------------------------------------
```

Comment:
env var might be cleaner unless you're wanting this to toggle on/off in a hot reload